Assignment9/Assignment9/RelayCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Windows.Input;
+
+namespace Assignment9
+{
+    public class RelayCommand : ICommand
+    {
+        public event EventHandler CanExecuteChanged;
+        public void RaiseCanExecuteChanged() => CanExecuteChanged?.Invoke(this, EventArgs.Empty);
+        public bool CanExecute(object parameter) => CanExecuteDelegate.Invoke();
+        public void Execute(object parameter) => ExecuteDelegate.Invoke();
```

Comment:
There is a warning generated here because the parameter types don't match the nullability of the interface.
```suggestion
        public bool CanExecute(object? parameter) => CanExecuteDelegate.Invoke();
        public void Execute(object? parameter) => ExecuteDelegate.Invoke();
```